apps/win32/.gitignore

```diff
@@ -67,3 +67,7 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
+
+
+# Build targets
+!dist
```

Comment:
I think we want this in the .npmignore not the .gitignore.

If we put it here, we'll end up checking in the bundle, when we just want to make sure it's included as part of package-publish.

Reply:
Ah, true. Thanks for catching that!

Reply:
I copied the contents from .gitignore over to .npmignore because I want to be sure it doesn't publish something we don't want but I'm unsure if I need to include anything from the .gitignore at the root of the repo.